Project.toml

```diff
@@ -5,15 +5,22 @@ version = "0.0.1"
 
 [deps]
 CDDLib = "3391f64e-dcde-5f30-b752-e11513730f60"
+COSMO = "1e616198-aa4e-51ec-90a2-23f7fbd31d8d"
```

Comment:
We don't want to add these in the dependency, they are only needed for tests

Reply:
Oups, removed them...